### DISCUSSION TITLE
FPS are shown in red with VSync enabled

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#19316] [Q&A] [answered]
As fps limiter I use VSync, but for some reason, lazer thinks 75fps is wrong: it's shown in red.
![image](https://user-images.githubusercontent.com/43073074/180597576-eaf00561-a766-46fd-89f4-fa95301dcddc.png)

The funny thing is that the counter is green when the app has no focus:
![image](https://user-images.githubusercontent.com/43073074/180597609-651cbc78-0f87-46c0-a653-c8e4869d9099.png)

**Comment by frenzibyte** [comment]
~~The counter being red means the game is expecting more on VSync (try hovering on the FPS counter and see what's the maximum number there)~~, it turns green when focus is lost because the game enforces all threads to 60fps, which your system is capable to operate at.

Can you also mention what's your monitor refresh rate?

**Comment by PandaDriver156** [comment]
I experience the same if VSync is enabled. Could it be caused by max fps being 1000fps?
(my refresh rate is 165hz)
![image](https://user-images.githubusercontent.com/46034537/180598141-6e829ece-bfea-4b66-a78c-2a945b360b98.png)

**Reply by peppy** [reply]
I believe we aren't considering vsync correctly at the moment, will look into this.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/19318
